platform/linuxbsd/detect.py

```diff
@@ -310,17 +310,16 @@ def configure(env):
 
     if os.system("pkg-config --exists alsa") == 0:  # 0 means found
         print("Enabling ALSA")
+        env["alsa"] = True
```

Comment:
Should we make it an option like pulseaudio so that users can choose not to compile alsa support in?

Reply:
Though that's maybe outside the scope of this PR. I suspect that alsamidi might fail compile if we disable alsa.